source/vision/visionHandler.py

```diff
@@ -99,10 +99,30 @@ def postGuiInit(self) -> None:
 
 	_allProviders: List[providerInfo.ProviderInfo] = []
 
+	def _getBuiltInProviderIds(self):
+		from visionEnhancementProviders.NVDAHighlighter import NVDAHighlighterSettings
+		from visionEnhancementProviders.screenCurtain import ScreenCurtainSettings
+		return [
+			NVDAHighlighterSettings.getId(),
+			ScreenCurtainSettings.getId()
+		]
+
 	def _updateAllProvidersList(self):
-		self._allProviders = list(_getProvidersFromFileSystem())
-		# Sort the providers alphabetically by name.
-		self._allProviders.sort(key=lambda info: info.displayName.lower())
+		# Sort the providers alphabetically by id.
+		# id is used because it will not vary by locale
+		all = sorted(

```

Comment:
I'd avoid the use of all here, even when it's a local

```suggestion
		allProviders = sorted(

```